src/sessions/softphone-session-handler.ts

```diff
@@ -61,8 +61,8 @@ export class SoftphoneSessionHandler extends BaseSessionHandler {
 
     this.log('debug', 'persistent connection event', event.eventBody);
 
-    if (event.eventBody.errorInfo) {
-      createAndEmitSdkError.call(this.sdk, SdkErrorTypes.call, event.eventBody.errorInfo.userMessage, event.eventBody.errorInfo);
+    if (event.eventBody.errorInfo && event.eventBody.persistentState === 'FAILED') {
+      createAndEmitSdkError.call(this.sdk, SdkErrorTypes.call, event.eventBody.errorInfo.userMessage, event.eventBody);
```

Comment:
Was the previous error a recent change or was it longstanding? If it was longstanding this could technically be a breaking change.

Reply:
Also, could this not be handled in the consuming app? I'm hesitant to want to change the errors we send up, we shouldn't particularly care about them. I think the consuming app should just filter out what it doesn't care about. It should just be our responsibility to pass them all up, and they can make an educated decision with the info we give.

Reply:
To your first point/question, what do we consider "longstanding"?  I believe this was introduced with Garrett's v10 changes in this regard.  Would that be considered a breaking change in this scenario?

Reply:
Can you update the PR name now that you've removed the failed state stuff?